test/functions_math.jl

```diff
@@ -154,3 +154,36 @@ end
     @test nda * inv(nda) ≈ NamedDimsArray{(:a, :a)}([1.0 0; 0 1])
     @test inv(nda) * nda ≈ NamedDimsArray{(:b, :b)}([1.0 0; 0 1])
 end
+
```

Comment:
Put all of the below into a `cov/cor` testset

Reply:
done :)